clients/client/src/main/java/org/projectnessie/client/ClientTreeApi.java

```diff
@@ -117,14 +117,15 @@ public Branch getDefaultBranch() {
 
   @Override
   public LogResponse getCommitLog(
-      String ref, Integer maxRecords, String pageToken, String queryExpression)
+      String ref, String hashOnRef, Integer maxRecords, String pageToken, String queryExpression)
```

Comment:
Guess this should be `startHash` instead of `hashOnRef`. Since we're already changing this, can you add `endHash` as welll?:

Reply:
I would rather prefer to have this in a separate commit/PR since it's a slightly different feature